translator/totomlconfig/sampleConfig/complete_linux_config.conf

```diff
@@ -153,6 +153,8 @@
     role_arn = "metrics_role_arn_value_test"
     rollup_dimensions = [["ImageId"], ["InstanceId", "InstanceType"], ["d1"], []]
     tagexclude = ["host", "metricPath"]
+    [outputs.cloudwatch.drop_original_metrics]
```

Comment:
Why is linux being changed?

Reply:
Changes propagated from https://github.com/aws/amazon-cloudwatch-agent/pull/357

Reply:
Removed.